docs/marketplace/dev/glue-api-guides/202108.0/searching-the-product-catalog.md

```diff
@@ -6751,7 +6751,454 @@ To search by products, send the request:
+                            },
+                            {
+                                "nodeId": 3,
+                                "name": "Camcorders",
+                                "docCount": 0,
+                                "children": []
+                            }
+                        ]
+                    },
+                    {
+                        "nodeId": 11,
+                        "name": "Telecom & Navigation",
+                        "docCount": 0,
+                        "children": [
+                            {
+                                "nodeId": 12,
+                                "name": "Smartphones",
+                                "docCount": 0,
+                                "children": []
+                            }
+                        ]
+                    },
+                    {
+                        "nodeId": 15,
+                        "name": "Cables",
+                        "docCount": 0,
+                        "children": []
+                    },
+                    {
+                        "nodeId": 16,
+                        "name": "Food",
+                        "docCount": 0,
+                        "children": [
+                            {
+                                "nodeId": 18,
+                                "name": "Fish",
+                                "docCount": 0,
+                                "children": []
+                            },
+                            {
+                                "nodeId": 19,
+                                "name": "Vegetables",
+                                "docCount": 0,
+                                "children": []
+                            }
+                        ]
+                    },
+                    {
+                        "nodeId": 9,
+                        "name": "Smart Wearables",
+                        "docCount": 0,
+                        "children": [
+                            {
+                                "nodeId": 10,
+                                "name": "Smartwatches",
+                                "docCount": 0,
+                                "children": []
+                            }
+                        ]
+                    }
+                ]
+            },
+            "links": {
+                "self": "https://glue.de.69-new.demo-spryker.com/catalog-search"
+            },
+            "relationships": {
+                "abstract-products": {
+                    "data": [
+                        {
+                            "type": "abstract-products",
+                            "id": "001"
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "links": {
+        "self": "https://glue.de.69-new.demo-spryker.com/access-tokens?q=001&include=abstract-products,concrete-products,product-offers",
```

Comment:
In this code sample, please replace the links everywhere with https://glue.mysprykershop.com